content.js

```diff
@@ -59,16 +59,13 @@
 
     return {
       getProductUrl: () => productLink.getAttribute('href'),
-

```

Comment:
As a personal preference I like to separate "unrelated" functions with a blank line

Reply:
Ok no problem, I see you fixed that yourself 👍